python/lsst/afw/display/interface.py

```diff
@@ -230,9 +252,11 @@ def __str__(self):
     # Handle Displays, including the default one (the frame to use when a user specifies None)
     #
     @staticmethod
-    def setDefaultBackend(backend):
+    def setDefaultBackend(backend, *args, **kwargs):
```

Comment:
Since you're changing `setDefaultBackend` to not only set the default backend but also to set default arguments for it, doesn't symmetry suggest you also change `getDefaultBackend` to return those arguments as well?